src/main/java/cubicchunks/asm/mixin/core/common/MixinWorldServer.java

```diff
@@ -106,7 +108,7 @@
 		for (int cubeX = spawnCubeX - spawnDistance; cubeX <= spawnCubeX + spawnDistance; cubeX++) {
 			for (int cubeZ = spawnCubeZ - spawnDistance; cubeZ <= spawnCubeZ + spawnDistance; cubeZ++) {
 				for (int cubeY = spawnCubeY + spawnDistance; cubeY >= spawnCubeY - spawnDistance; cubeY--) {
-					serverCubeCache.loadCube(cubeX, cubeY, cubeZ, LOAD_OR_GENERATE);
+					serverCubeCache.getCube(new CubeCoords(cubeZ, cubeY, cubeZ), IProviderExtras.Requirement.LIGHT);
```

Comment:
Allocating a new object here for every call is going to wreck performance and blow up memory use. Either use something pooled or (preferably, in my opinion), just use x,y,z directly